### PR TITLE
test(integration-karma): add tests for updating class attribute

### PR DESCRIPTION
**Pull request by nolanlawson**
## Details

While working on patch flags, I realized we were missing some tests for updating dynamic `class` attributes with null/undefined/empty string. This borrows some of the same tests from `integration-karma/test/template/attribute-style/index.spec.js` to accomplish this.

## Does this pull request introduce a breaking change?

* ✅ No, it does not introduce a breaking change.

## Does this pull request introduce an observable change?

* ✅ No, it does not introduce an observable change.

### CODE REVIEW

File: packages/@lwc/integration-karma/test/template/attribute-class/index.spec.js [hunk 1]
```diff
@@ -77,4 +77,44 @@ describe('dynamic class attribute', () => {
...
+                expect(elm.shadowRoot.querySelector('div').getAttribute('class')).toBe(
+                    expectedValue
+                );
+            });
+        }
+
+        testUpdateClassAttribute('null', null, '');
+        testUpdateClassAttribute('undefined', undefined, '');
+        testUpdateClassAttribute('empty string', '', '');
+        testUpdateClassAttribute('class string', 'my-class', 'my-class');
```
Comment (nolanlawson) [inline packages/@lwc/integration-karma/test/template/attribute-class/index.spec.js:118]: Frankly this doesn't make a lot of sense (you get `null` at first, and then `''` after an update), but I'm just trying to test the existing behavior.

**Review by ekashida — APPROVED** [review]

**Review by jmsjtu — APPROVED** [review]
